Shared/Samples/Authenticate with OAuth/README.md

```diff
@@ -0,0 +1,39 @@
+# Authenticate with OAuth
+
+Authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers).
+
+![Login screen](authenticate-with-oauth1.png)
+![Map view after authentication](authenticate-with-oauth2.png)
+
+## Use case
+
+Your app may need to access items that are only shared with authorized users. For example, your organization may host private data layers or feature services that are only accessible by verified users. You may also need to take advantage of premium ArcGIS Online services, such as geocoding or routing, that require a named user login.
+
+## How to use the sample
+
+When you run the sample, the app will load a web map which contains premium content. You will be challenged for an ArcGIS Online login to view the private layers. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). If you authenticate successfully, the traffic layer will display, otherwise the map will contain only the public basemap layer.
```

Comment:
```suggestion
Upon launch, a web map containing premium content will load. You will be prompted by an ArcGIS Online login to view the private layers. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). If you authenticate successfully, the traffic layer will display; otherwise, the map will contain only the public basemap layer.
```
Just my opinions, feel free to pick and choose from my suggestions.

Reply:
I'm ok with the change but this is same as in iOS sdk sample.